contributing/data-model-schema.txt

```diff
@@ -29,9 +29,32 @@ wastes your time. From the top-level :file:`bioformats` folder, while
 the most thorough is :command:`git clean -dfx`.
 
 
+Major or minor release?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A minor release of the OME model schema may suffice for changes like
```

Comment:
Yes I do not think we have used it extensively across our documentation yet. An example of usage is at the top of https://github.com/openmicroscopy/design/issues/55. 

Happy to discuss if this is still premature but we could certainly promote a more general usage  /cc @hflynn @rleigh-codelibre